src/tabset/tabset.ts

```diff
@@ -19,27 +21,43 @@ export class NgbTabContent {
   constructor(public templateRef: TemplateRef<any>) {}
 }
 
+/**
+ * A directive representing an individual tab.
+ */
 @Directive({selector: 'ngb-tab'})
 export class NgbTab {
+  /**
+   * Unique tab identifier. Must be unique for the entire document for proper accessibility support.
+   */
+  @Input() id: string = `ngb-tab-${nextId++}`;
```

Comment:
we actually need it, at least now, for API docs documentation (that is, when extracting API docs I don't know how to infer types). Will update #222